ethereum/migrations/prod/9_pyth_enable_governance.js

```diff
@@ -0,0 +1,23 @@
+require('dotenv').config({ path: "../.env" });
+
+const PythUpgradable = artifacts.require("PythUpgradable");
+const governanceChainId = process.env.GOVERNANCE_CHAIN_ID;
+const governanceEmitter = process.env.GOVERNANCE_EMITTER;
+
+console.log("governanceEmitter: " + governanceEmitter);
+console.log("governanceChainId: " + governanceChainId);
+
+const { upgradeProxy } = require("@openzeppelin/truffle-upgrades");
+
+/**
+ * Version 1.0.0
+ * 
+ * This change:
+ * - Moves away single ownership to Governance coming from the Wormhole
+ */
+module.exports = async function (deployer) {
+    const proxy = await PythUpgradable.deployed();
+    await upgradeProxy(proxy.address, PythUpgradable, { deployer, unsafeSkipStorageCheck: true });
+    await proxy.updateGovernanceDataSource(governanceChainId, governanceEmitter, 0);
+    await proxy.renounceOwnership();
```

Comment:
i suggest renouncing ownership as a separate migration. Deploy the WH control changes, then run a WH governance instruction to test that it works, then renounce ownership after. It's a bit more forgiving that way in case we screw something up.

Reply:
Yeah, I was planning to do them separately to test. But It's good to have them in separate migration files (make it easier to test separately)